docs/glossary.rst

```diff
@@ -180,3 +180,7 @@ Glossary
     Workload
       A database category that allows you to manage different sources of database activities, such as requests or administrative activities. By defining workloads, you can specify how ScyllaDB will process those activities. For example, you can prioritize one workload over another (e.g., user requests over administrative activities). See :doc:`Workload Prioritization </using-scylla/workload-prioritization>`.
 
+    MurmurHash3
+       A hash function `created by Austin Appleby <https://en.wikipedia.org/wiki/MurmurHash>`_, and used by the :term:`Partitioner` to distribute the partitions between nodes. 
+       The MurmurHash3 version used in ScyllaDB is originated from `Apache Cassandra <https://commons.apache.org/proper/commons-codec/apidocs/org/apache/commons/codec/digest/MurmurHash3.html>`_.
```

Comment:
Maybe you want to common that Cassandra's variant is NOT identical to the official MurmurHash3 calculation. It is admitted in
https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/utils/MurmurHash.java#L31-L33
that this is a bug, that can't be fixed. The sign problem is explained in https://github.com/russss/murmur3-cassandra.

Reply:
Good point. See latest version